wgpu-native/Cargo.toml

```diff
@@ -17,15 +17,15 @@ crate-type = ["lib", "cdylib", "staticlib"]
 
 [features]
 default = []
-remote = []
+local = ["winit", "gfx-backend-empty/winit"]
```

Comment:
I think it would still be good to have `winit` as a feature separately - there could be `local` with `winit`, `local` with `sdl`, etc.